recipes/graylog-forwarder/files/pre-install

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+set -e
+
+user="graylog-forwarder"
+group="graylog-forwarder"
+datadir="/usr/share/graylog-forwarder"
```

Comment:
Same as in `post-install`. Let's use `/var/lib/graylog-forwarder` here.

Reply:
We're still putting the jars into /usr/share, though, right? So we still have to create that directory. Unless you mean to put all the static files into /var/lib, too.

Reply:
Yep, the JARs still go into `/usr/share/graylog-forwarder`. But we don't need to manually create that directory because the package system takes care of that.

Reply:
This is fixed.